apps/server/src/modules/tldraw/service/tldraw.ws.service.ts

```diff
@@ -183,11 +183,15 @@ export class TldrawWsService {
 		doc.connections.set(ws, new Set());
 
 		ws.on('error', (err) => {
-			this.logger.warning(new WebsocketErrorLoggable(err));
+			this.logger.warning(new WebsocketErrorLoggable('onError', err));
 		});
 
 		ws.on('message', (message: ArrayBufferLike) => {
-			this.messageHandler(ws, doc, new Uint8Array(message));
+			try {
+				this.messageHandler(ws, doc, new Uint8Array(message));
+			} catch (err) {
+				this.logger.warning(new WebsocketErrorLoggable('onMessage', err));
```

Comment:
This catch is fine, but please remove the try catch in messageHandler(). Otherwise you do it twice.

Reply:
fixed